backend/routes.py

```
@@ -828,9 +1255,279 @@ def run_manual_cleanup(dry_run: bool = True):
+
+        # Get and validate max_age_days
+        max_age_days = 30  # Conservative default
+
+        if data and "max_age_days" in data:
+            try:
+                max_age_days = int(data["max_age_days"])
+                max_age_days = validate_max_age_days(max_age_days)
+            except ValueError:
+                raise HTTPException(
+                    status_code=400, detail="max_age_days must be a valid integer."
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)